README.md

```diff
@@ -34,7 +34,8 @@ Application Options:
   -i, --input=FILE         The input video filename
   -o, --output=FILE        The output video filename (default: output.mp4)
   -b, --bitrate=BITRATE    The bitrate in bytes/second to encode in. If not specified, take the same bitrate as the input file
+  -s, --squeeze            Squeeze 4:3 video stretched to 16:9 (e.g. Caddx Tarsier 2.7k60)
```

Comment:
The description seems to be inconsistent with the one in `superview.go`, can you fix that?

Reply:
Sure, fixed that.